FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift

```diff
@@ -113,4 +118,57 @@ class FirefoxPrivateNetworkVPNUITests: BaseTestCase {
         waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
         XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
         }
-    }
+
+    func testSignInAsSubscribedUser() {
+            // The main screen is shown
+            waitForExistence(app.staticTexts["Mozilla VPN"], timeout: 15)
+            XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists, "The main page is not loaded correctly")
+
+            // Tap on Get started
+            app.buttons["Get started"].tap()
+            waitForExistence(app.staticTexts["Use a different account"], timeout: 15)
```

Comment:
I would add a comment here // Removing previous account or something, you are saying Tap on Get started, but then what follows is not very clear